twilight-gateway/src/stream.rs

```diff
@@ -32,9 +32,7 @@
 //! [gateway-parallel]: https://github.com/twilight-rs/twilight/blob/main/examples/gateway-parallel.rs
```

Comment:
Look over the module documentation for the closure. Add note about all items in this module can be user implemented / don't use privileged, private, APIs.